stdlib/public/Concurrency/TaskSleepDuration.swift

```diff
@@ -142,4 +142,21 @@ extension Task where Success == Never, Failure == Never {
   ) async throws {
     try await clock.sleep(until: deadline, tolerance: tolerance)
   }
+  
+  /// Suspends the current task for the given duration on a continuous clock.
+  ///
+  /// If the task is cancelled before the time ends, this function throws 
+  /// `CancellationError`.
+  ///
+  /// This function doesn't block the underlying thread.
+  ///
+  ///       try await Task.sleep(for: .seconds(3))
+  ///
+  /// - Parameter duration: The duration to wait.
+  @available(SwiftStdlib 5.7, *)
+  public static func sleep(for duration: Duration) async throws {
+    for duration: Duration
+  ) async throws {
```

Comment:
```suggestion
```

Sorry, my suggestion couldn't be applied literally as it didn't have the right lines highlighted. This should do it.